src/node.cc

```diff
@@ -17,6 +17,7 @@
 
 #if defined(NODE_HAVE_I18N_SUPPORT)
 #include "node_i18n.h"
+#include <unicode/putil.h>
```

Comment:
Actually it should be uvernum.h and not uversion.h to get only the #define
and not additional functions.